### DISCUSSION TITLE
Does partner sharing share albums?

### DISCUSSION THREAD

**Discussion by andrejohansson** [immich-app/immich#3620] [Q&A] [answered]
I have a question regarding partner sharing:

- I have created normal albums (non-shared)
- I have shared all my images with my parents

Result:
- They can see all my photos via going into partner photos timeline
- They **cannot** see my albums

Is this the intended functionality of partner sharing today? 
Is there any way for me to let my partner see all my albums and not only my unorganized photos?

**Comment by jrasm91** [comment, marked as answer]
Currently partner sharing does exactly that - share the main timeline page.

I don't believe it is planned to include partner albums. One option would be to share all the albums with the user though. In the future we might make it easier to do this in bulk, but at the moment it can only be done one at a time
